Resources/FAQ/Jobs-Discord-Asked-Questions.md

```diff
@@ -110,7 +142,7 @@ To get the best help you can get we need information, then Zrips on Github, or t
 **# Economy manager:** `ver CMI`
 **# Permission Manager:** `ver LuckPerms`
 
-- Also keep an eye on the console/logs, to make sure you spot any console errors during startup of any plugins, and/or when you try to do something with this plugin. If you want to paste your big error msgs, please use a pastebin like service like <https://paste.md-5.net/>
+- Keep an eye on the console or check the logs to make sure you spot any errors during the startup of any plugins. If you see an error, try to disable the plugin causing it and retry. If there's a need to paste extensive logs and error messages, use a Pastebin type service such as  you want to paste your big error msgs, please use a text storage site like [Hastebin](https://paste.md-5.net/)
```

Comment:
The advice of disabling the plugin causing the error isn't great, as it may not be the root cause, and only a symptom.
Instead, advising people to read the root cause of the error might be helpful if it is solely user caused.
Tho if they are on latest, it should still be reported